studio/schemas/documents/sections/keyBenefits.js

```diff
@@ -0,0 +1,45 @@
+import { MasterDetailIcon } from '@sanity/icons'
+import supportedLanguages from '../../supportedLanguages';
+
+
+const baseLanguage = supportedLanguages.find(l => l.isDefault);
+
+export default {
+  name: 'keyBenefits',
+  type: 'document',
+  title: 'Key Benefits',
+  icon: MasterDetailIcon,
```

Comment:
Please, change the icon to SplitHorizontalIcon

Reply:
done